instrumentation-core/spring/spring-boot-autoconfigure/spring-boot-autoconfigure.gradle

```diff
@@ -18,21 +18,27 @@ bootJar {
 }
 
 dependencies {
-  annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
+  annotationProcessor "org.springframework.boot:spring-boot-autoconfigure-processor"
+  implementation "org.springframework.boot:spring-boot-starter-validation"
   
   testImplementation('org.springframework.boot:spring-boot-starter-test') {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
   
-  implementation 'org.springframework.boot:spring-boot-starter-web'
   implementation project(':instrumentation-core:spring:spring-webmvc-3.1')
   implementation project(':instrumentation-core:spring:spring-web-3.1')
-  
-  implementation 'org.springframework.boot:spring-boot-starter-webflux'
   implementation project(':instrumentation-core:spring:spring-webflux-5.0')
   
   api deps.opentelemetryApi 
-  api "io.opentelemetry:opentelemetry-exporters-logging:0.5.0"
+
+  compileOnly "io.opentelemetry:opentelemetry-exporters-logging:0.5.0"
+  compileOnly deps.opentelemetryJaeger
+  compileOnly deps.opentelemetryOtlp
+  compileOnly group: 'io.grpc', name: 'grpc-api', version: '1.24.0'
```

Comment:
It sounds like the transitive dependency problem got fixed. Doesn't it work to remove these extra `compileOnly` completely?